vendor/bat-native-ledger/src/bat_helper_unittest.cc

```diff
@@ -35,3 +35,65 @@ TEST(BatHelperTest, isProbiValid) {
       "100000000000000000010000000000000000001000000000000000000");
   ASSERT_EQ(result, false);
 }
+
+TEST(BatHelperTest, HasSameDomainAndPath) {
+  std::string url("https://k8923479-sub.cdn.ttvwn.net/v1/segment/");
+  std::string url_portion(".ttvwn.net");
```

Comment:
should be just `ttvwn.net` instead of `.ttvwn.net`? same for instances below

Reply:
That was intentional. Should we not test for leading `.`?

Reply:
i only saw it called without the leading dot in https://github.com/brave/brave-core/pull/1792/files#diff-501b5889b2700e1f1a74c584464a569cR39. is there a need for this to be called with a leading dot?

'domain' usually refers to a string like `example.com` or `com`, without a leading dot

Reply:
Prior to this pr, the leading . Was there but I suppose it’s no longer needed. I’ll remove it.

Reply:
removed